src/renderer/js/redux/reducers/app.js

```diff
@@ -1,22 +1,45 @@
-import * as types from "constants/action_types";
-import * as modalTypes from "constants/modal_types";
+// @flow
 
+import * as types from "../../constants/action_types";
+import * as modalTypes from "../../constants/modal_types";
+
+// $FlowIssue: Flow cannot find election
 const { remote } = require("electron");
+
 const application = remote.app;
 const win = remote.BrowserWindow.getFocusedWindow();
 
 const reducers = {};
-const defaultState = {
+type appState = {
```

Comment:
thoughts on all upper case for types?

Reply:
(also, state types will generally need to be exported so they can be used by selectors)

Reply:
Also also, this should be a sealed object.

Reply:
I agree, I meant to bring this up, we also need to figure out how to properly do naming.